### DISCUSSION TITLE
any plans to build a cli to use this tooling?

### DISCUSSION THREAD

**Discussion by PankajMoolrajani** [oxsecurity/codetotal#116] [General]
i like that all the scanners are packed inside one wrapper. It will be really powerful to expose this via cli so that it can be used in ci/cd pipeline easily. 
Currently i see the only way to use it is via web interface which is good for just testing out the tool.

**Comment by echoix** [comment]
Have you ever tried https://megalinter.io/latest/ https://github.com/oxsecurity/megalinter/? This is what powers CodeTotal; you should take a look at all the integration that is done already for CI uses with the multiple different reporters and flexible configuration to fit very diverse project types!

**Reply by nvuillam** [reply]
@PankajMoolrajani MegaLinter Security flavor is behind CodeTotal, so as @echoix said you can easily use it on any CI platform (GitHub, gitlab, azure, Bitbucket, jenkins...)

https://megalinter.io/latest/flavors/security/

MegaLinter can also be run locally via command line -> https://megalinter.io/latest/mega-linter-runner/

And if you want a more advanced securisation of your software supply chain, you can alse have a look at [OX Security](https://ox.security) :)
